modules/api/src/main/java/org/apache/fluo/api/config/FluoConfiguration.java

```diff
@@ -510,6 +514,19 @@ public String getAccumuloPassword() {
     throw new NoSuchElementException(ACCUMULO_PASSWORD_PROP + " is not set!");
   }
 
+  public FluoConfiguration setAccumuloAuthorizations(String authsCsv) {
+    setProperty(ACCUMULO_AUTH_PROP, authsCsv);
+    return this;
+  }
+
+  public String getAccumuloAuthorizations() {
```

Comment:
This could return a list.

```suggestion
  /**
    *  Returns the authorizations configured for reading data or an empty list if none are configured.
    *
    * @since 1.2.0
    */
  public List<String> getAccumuloAuthorizations() {
```